src/System.Runtime.Extensions/tests/TestApp/TestApp.csproj

```diff
@@ -18,8 +18,5 @@
       <Link>Common\System\Diagnostics\CodeAnalysis\ExcludeFromCodeCoverageAssemblyAttribute.cs</Link>
     </Compile>
   </ItemGroup>
-  <ItemGroup>
-    <None Include="project.json" />
```

Comment:
Yeah, I was originally just looking for `TargetingPackReference` and deleted these if they were in the same file. I will go through again and delete the rest of the unnecessary `project.json`s.